### ISSUE TITLE
Training on a device which doesn't support cuda 9? (> Cuda 11)

### ISSUE THREAD

**Issue by jamesheatonrdm** [nianticlabs/monodepth2#407]
Hi, I have seen this issue on another question but wanted to create my own thread in case other people are having this issue and can provide some insight.

I am on Ubuntu 18.02 using an NVIDIA 3060 GPU. The system requirements say that I should be using PyTorch 0.4.1 and CUDA 9.2 however these older versions are not compatible with this graphics card. If I try to install the recommended requirements I cannot send anything to the GPU. Calling tensor.to("cuda") causes the program to hang indefinitely because of the version mismatch.

Apparently using CUDA 11 with a newer version of PyTorch > 0.4.1 will result in issues with training and cause the loss to not converge. Therefore I would like to know if anyone has got around this problem.  Is there any way to have PyTorch 0.4.1 and Cuda 9.2 working with an NVIDA 3060? Or will I have to just use CPU training?

Any help is appreciated.

**Comment by zhouxf53** [comment]
I have no problem using
```
torch                  1.8.0+cu111
torchaudio             0.8.0
```
Very minimal changes to the code is needed

**Comment by jamesheatonrdm** [comment]
@zhouxf53  Thanks for letting me know, could you please tell me which parts of the code I need to change? It can run fine when I am using torch 1 and cuda 11 it is just I am having issues with the loss converging and have been told that this is due to the versions

**Comment by zhouxf53** [comment]
> @zhouxf53 Thanks for letting me know, could you please tell me which parts of the code I need to change? It can run fine when I am using torch 1 and cuda 11 it is just I am having issues with the loss converging and have been told that this is due to the versions

I changed the augmentation part since `transforms.ColorJitter` behavior has been changed. I do not have the convergence issue. Since I was trained on my own dataset, my experience may not be applicable to you.

**Comment by songlin** [comment]
@zhouxf53  Can you share the `torchvision` version you use? and can you succesfully reproduce the paper's result?

**Comment by zhouxf53** [comment]
> @zhouxf53 Can you share the `torchvision` version you use? and can you succesfully reproduce the paper's result?

`torchvision                  0.9.0+cu111`
I trained with my own data to eval absRel, I did not follow through on KITTI training to the very end
